src/gc/gc.cpp

```diff
@@ -34781,7 +34781,11 @@ HRESULT GCHeap::Initialize()
         return CLR_E_GC_BAD_AFFINITY_CONFIG;
     }
 
-    if ((cpu_index_ranges_holder.Get() != nullptr) || (config_affinity_mask != 0))
+    if ((cpu_index_ranges_holder.Get() != nullptr)
+#ifndef FEATURE_PAL
```

Comment:
Should this rather be `PLATFORM_WINDOWS` ? Note that `FEATURE_PAL` is not defined for standalone GC.

Reply:
You are right - and we have yet another FEATURE_PAL ifdef in the gc that should get the same treatment.